bookwyrm/tests/importers/test_importer.py

```diff
@@ -65,25 +65,28 @@ def test_create_job(self, *_):
 
         import_items = models.ImportItem.objects.filter(job=import_job).all()
```

Comment:
This seems to work but it is a bit whacky. Would it work better/as well to just order the queryset and keep the existing asserts?

```py
import_items = models.ImportItem.objects.filter(job=import_job).all().order_by("id")
``

Reply:
oh yeah that is way simpler!